conf/schema/natural.json

```diff
@@ -102,6 +102,15 @@
             { "name": "natural=mountain", "weight": 0.8 }
         ]
     },
+    {
+        "geometries": [
+            "node"
+        ],
+        "categories": [],
+        "isA": "natural",
+        "name": "natural=tree",
+        "objectType": "tag"
+    },
```

Comment:
@mattjdnv This change is in the grail branch.  It may or may not be causing some test failures.  Is this something we do want merged?